src/Indice.Features.Cases.App/src/app/shared/ajsf/json-schema-frameworks/tailwind-framework/date-widget/date-widget.component.ts

```diff
@@ -39,4 +52,27 @@ export class DateWidgetComponent implements OnInit {
     this.jsf.updateValue(this, event.target.value);
   }
 
+  private setMinDate(options: any): string {
+    var currentDate = moment(new Date());
+    currentDate.add(options.minDateYears ?? this.minDateYears, 'years').year();
+    currentDate.add(options.minDateMonths ?? this.minDateMonths, 'months').month();
+    currentDate.add(options.minDateDays ?? this.minDateDays, 'days').date();
+
+    var year = currentDate.year();
```

Comment:
αντι για ολο αυτο, μηπως θες κατι τετοιο 

`return currentDate.format("YYYY-MM-DD")`

check: https://momentjs.com/docs/#/displaying/format/

Reply:
To μόνο πρόβλημα που μπορώ να φανταστώ με την συγκεκριμένη υλοποίηση είναι ότι οι μήνες για κάποιο λόγο ξεκινάνε απ το 00 και φτάνουν μέχρι το 11 και θες να κάνεις την πατέντα με το +1. Θα δοκιμάσω όμως να δω αν με το formatting φτιάχνεται αυτό!

Reply:
Αυτο που λες ισχύει, αλλά εκει βοηθάει η `format` του momemntjs. Αφου την βαλαμε, ας την αξιοποιησουμε. Εχει καλο doc επίσης

![image](https://user-images.githubusercontent.com/8275452/211564262-4bfdbb3e-775d-4850-8cb9-3dd6e8901a09.png)

Βέβαια, τωρα κοιταω μηπως δεν χρειαζεται τελικα, για να μην βαρύνει το τελικο bundle σε MB

Reply:
Το αρχικό πρόβλημα που έλυσε (και κατ' επέκταση χάρη στο moment γλιτώσαμε και αρκετές γραμμές κώδικα) είναι πως σου υπολόγιζε με βάση τις ημέρες πριν ή μετά αν άλλαζε και ο μήνας ή ο χρόνος και δεν χρειαζόταν να κάνεις τον έλεγχο με το χέρι. Αν βρεθεί εναλλακτική που να συμφέρει περισσότερο σε θέμα bundle μου λες και το αλλάζουμε.